lib/egg.js

```diff
@@ -40,7 +40,7 @@ class EggApplication extends EggCore {
      */
     this.messenger = new Messenger();
 
-    this.dumpConfig();
+    this.beforeStart(() => this.dumpConfig());
```

Comment:
Should dump after all beforeStart hooks executed.

Maybe `afterStart()` hook. cc @popomore